.env.sample

```diff
@@ -3,9 +3,6 @@
 # as this file is stored as plain text on disk,
 # and is therefore not secure enough.
 
-# Name
-YOUR_NAME=
-
```

Comment:
This is replicated in https://github.com/hedera-dev/hello-future-world-js/pull/8